legate/tester/stages/test_stage.py

```diff
@@ -252,12 +252,12 @@ def run(
 
         cov_args = self.cov_args(config)
 
-        cmd = [str(config.legate_path)] + cov_args + [str(test_path)]
+        cmd = [str(config.legate_path)] + cov_args
 
         stage_args = self.args + self.shard_args(shard, config)
         file_args = self.file_args(test_file, config)
 
-        cmd += stage_args + file_args + config.extra_args
+        cmd += stage_args + [str(test_path)] + file_args + config.extra_args
```

Comment:
Do we need to be careful about what we put in file/custom args now? Since the legate args are only read before the script name?

Reply:
My only previous experience and expectation for  `file_args` was as args that the example script itself would be parsing e.g. 

https://github.com/nv-legate/cunumeric/blob/12233e8021be8d490e1b87097037be2396e8ba3b/examples/cholesky.py#L35-L54

If we need to supply per-file CLLR args then yes we would need to explicitly provide those in another way (or else hold off on the before/after splitting)

Reply:
@manopapad Currently, the below issue happened. The WAR is to rename /opt/conda/envs/legate/bin/coverage as coverage.py.

```
+LEGATE_TEST=1 legate /opt/conda/envs/legate/bin/coverage run -a --branch /opt/legate/cunumeric/tests/integration/test_take_along_axis.py -cunumeric:test --fbmem 4096 --gpus 1 --gpu-bind 0 -v
[FAIL] (GPU) tests/integration/test_take_along_axis.py (exit: 1)
   ERROR: usage: --capture=sys [options] [file_or_dir] [file_or_dir] [...]
   --capture=sys: error: unrecognized arguments: -a --branch
     inifile: /opt/legate/cunumeric/pyproject.toml
     rootdir: /opt/legate/cunumeric (edited)
```